assets/javascripts/audit_log.js

```diff
@@ -4,6 +4,42 @@
 var audit_url;
 var ajax_url;
 
+function getURLForType(type, event_data) {
+    switch (type) {
+        case 'jobtemplate_create':
+            if (event_data.job_group_id) {
+                return '/admin/job_templates/' + event_data.job_group_id;
+            }
+            break;
+        case 'job_create':
+        case 'job_update_result':
+        case 'job_done':
+        case 'job_restart':
+            return '/tests/' + event_data.id;
+        case 'jobgroup_create':
+            if (event_data.id) {
+                return '/group_overview/' + event_data.id;
+            }
+            break;
+        case 'iso_create':
+            return 'admin/productlog?id=' + event_data.scheduled_product_id;
+        case 'table_create':
+            if (event_data.id) {
+                switch (event_data.table) {
+                    case 'Machines':
+                        return 'admin/machines?q=' + event_data.id;
+                    case 'Products':
+                        return 'admin/products?q=' + event_data.id;
```

Comment:
May I ask why we query the machine/product/test_suites by id? the `id` is not shown in table, and the value of `q` will be shown the search textbox, is it?

Reply:
We have no route showing a single product/machine/testsuite - but this depends on searching by ID being implemented indeed.